site/docs/spark-queries.md

```diff
@@ -243,6 +269,101 @@ Note:
 2. `contains_nan` could return null, which indicates that this information is not available from files' metadata. 
    This usually occurs when reading from V1 table, where `contains_nan` is not populated. 
 
+### All Manifests
+
+To show a table's valid file manifests and each file's metadata, run:
+
+!!! Note
+    a valid manifest file is one that is referenced from any snapshot currently tracked by the table.
+
+!!! WARNING
+    this table may return **duplicate** rows
```

Comment:
Documentation should use correct capitalization. In this case, sentences should start with a capital letter.

Reply:
Looks like there are several cases of sentences that don't start with capitals, so be sure to fix all of them, please!